surveys/micro/leadership-council-2024.md

```diff
@@ -0,0 +1,45 @@
+# Survey questions
+
+This is a quick annual survey for Rust-lang project members to provide feedback on the [Leadership Council](https://forge.rust-lang.org/governance/council.html). Your feedback is appreciated to help steer the Council activities. As always, if you have specific questions or feedback that you would like to provide, feel free to reach out to your Council representative.
+
+This survey is anonymous, though the Council will see the answers. A summary of these results will be released after the survey is complete.
```

Comment:
> This survey is anonymous, though the Council will see the answers

This wording seems a bit weird. Of course that the Council will see the answers (that sounds obvious), but this wording makes it sound to me like the Council will see *who* answered (since the second part of the sentence is tied to the anonymity remark).

Reply:
Happy for any suggestions to make that clearer. I think there were two points that seemed important to me: it is anonymous, and who will see the answers. Unless stated, I don't think it is obvious who will see the answers since an alternate solution would be to have a third party summarize the results and the Council only sees a summary.

If it would help, we could drop the word `though` and just make it two sentences. Maybe something like:

> This survey is anonymous. The Council will go through all answers and release a summary after the survey is complete.

Reply:
I think that's much better, thanks!